pkg/operator/controller/ingress_status_test.go

```diff
@@ -59,136 +68,117 @@ func provisionedLBservice(owner string) *corev1.Service {
 func clusterIPservice(name string) *corev1.Service {
 	return &corev1.Service{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "router-" + name + "internal",
+			Name: name + "-internal",
 		},
 		Spec: corev1.ServiceSpec{
 			Type: corev1.ServiceTypeClusterIP,
 		},
 	}
 }
 
-func cond(t string, status operatorv1.ConditionStatus, reason string) operatorv1.OperatorCondition {
-	return operatorv1.OperatorCondition{
-		Type:   t,
-		Status: status,
-		Reason: reason,
+func failedCreateLBEvent(service string) corev1.Event {
+	return corev1.Event{
+		Type:    "Warning",
+		Reason:  "CreatingLoadBalancerFailed",
+		Message: "failed to ensure load balancer for service openshift-ingress/router-default: TooManyLoadBalancers: Exceeded quot aof account",
```

Comment:
s/quot aof/quota of/

Reply:
Fixed... again (my fix for the original had a typo 🤦‍♀️)